Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM rust:1.59 as build
+
+# create a new empty shell project
+RUN USER=root cargo new --bin sol_to_uml
```

Comment:
why do you need `USER=root`?

Reply:
why not just mkdir? Or even workdir?

Reply:
USER=root comes from the guide I used to create this Dockerfile. As I can see it's not required, so I removed it. 'cargo new' is mainly used to create main.rs file in src cause I don't copy ./src in the first step but should run 'cargo build'.